css/selectors/invalid.json

```
@@ -51,7 +51,8 @@
         },
         "form": {
           "__compat": {
-            "description": "Applies to <a href='https://developer.mozilla.org/docs/Web/HTML/Element/form'>&lt;form&gt;</a> elements",
+            "mdn_url": "https://developer.mozilla.org/docs/Web/HTML/Element/form",
+            "description": "<code>&lt;form&gt;</code>",
```

I prefer the more verbose description text, though you're right to remove the link

```suggestion
            "description": "Applies to <code>&lt;form&gt;</code> elements",
```